pennylane/drawer/tape_mpl.py

```diff
@@ -191,17 +191,20 @@ def _get_measured_wires(measurements, wires) -> set:
 
 
 def _add_classical_wires(drawer, layers, wires):
```

Comment:
Potentially rename `layers` here to `cwire_layers` and `cwire_wires`.

Reply:
Done! Also renamed some inner variables to (hopefully) make things a little better.